frontend/__tests__/module/k8s/autocomplete.spec.ts

```diff
@@ -85,22 +85,20 @@ describe('getCompletions', () => {
   });
 
   it('invokes callback with appropriate completions for properties', (done) => {
-    const swagger = {
-      definitions: {
-        'io.k8s.api.apps.v1.Deployment': {},
-        'io.k8s.api.apps.v1.DeploymentSpec': {
-          properties: {
-            minReadySeconds: {
-              description: 'Dummy property',
-              type: 'integer',
-              format: 'int32',
-            },
+    const definitions = {
+      'io.k8s.api.apps.v1.Deployment': {},
+      'io.k8s.api.apps.v1.DeploymentSpec': {
+        properties: {
+          minReadySeconds: {
+            description: 'Dummy property',
+            type: 'integer',
+            format: 'int32',
           },
         },
       },
     };
     sessionMock.getLines = jasmine.createSpy('getLinesSpy').and.returnValue(['kind: Deployment', 'apiVersion: apps/v1']);
-    spyOn(window.localStorage, 'getItem').and.returnValue(JSON.stringify(swagger));
```

Comment:
I'm concerned this was passing CI

Reply:
It's a unit test.

Reply:
But we're spying on localStorage, not sessionStorage?

Reply:
Interestingly, whichever you spy on (`localStorage` or `sessionStorage`), it seems to work. Removing the spy entirely causes the tests to fail. So I guess it's OK? The tests are running and working.

Reply:
See `__mocks__/localStorage.ts`.

Reply:
Ah ha, that explains it. Thanks.